src/test/java/com/epam/reportportal/testng/BuildTestTest.java

```diff
@@ -98,7 +102,16 @@ public void testSkippedIssue() {
 		parameters.setSkippedAnIssue(true);
 		parameters.setAttributes(Sets.<ItemAttributesRQ>newHashSet());
 		StartLaunchRQ startLaunchRQ = testNGService.buildStartLaunchRq(parameters);
+		assertThat(startLaunchRQ.getAttributes().size(), Matchers.is(4));
 		assertTrue(startLaunchRQ.getAttributes().contains(itemAttributeResource));
+		final Set<String> keys = startLaunchRQ.getAttributes().stream().map(ItemAttributeResource::getKey).collect(toSet());
+		predefinedProperties.forEach((key, value) -> {
+			assertTrue(keys.contains(key));
+			startLaunchRQ.getAttributes().stream().filter(attr -> attr.getKey().equalsIgnoreCase(key)).forEach(it -> {
+				assertTrue(predefinedProperties.get(key).matcher(it.getValue()).matches());
+				assertTrue(it.isSystem());
+			});
+		});
```

Comment:
It's not OK to put everything into one test. Please keep them atomic, this means create a new test for that particular piece of functionality.